iib/web/api_v1.py

```diff
@@ -432,6 +433,30 @@ def get_builds() -> flask.Response:
                 )
             )
 
+        if from_index_startswith:
+            query_params['from_index_startswith'] = from_index_startswith
+            from_index_results = Image.query.filter(
```

Comment:
should this be `from_index_results` or `from_index_startswith_results `?

Reply:
yeah it would be better :)

Reply:
please review changes

Reply:
Thank you!